src/modules/RefreshmentMap/index.tsx

```diff
@@ -60,13 +63,31 @@ export const RefreshmentMap: FC<RefreshmentMapPropType> = (pageProps) => {
           {...TEMPERATURE_DATA}
           fillColorProperty={activeHour.vectorTilesetKey}
         />
-        <RasterLayer
-          id="shade-data"
-          url={activeHour.shadeTilesetId}
-          bounds={[13.06, 52.33, 13.77, 52.69]}
-          minZoom={14}
-          opacity={0.5}
-        />
+        {hourKeys.map((key) => {
+          const item = HOURS[key]
+          if (key !== activeHourKey) {
```

Comment:
This condition should only happen on the opacity prop otherwise you are repeating yourself too much. No drama though :)